src/objects/core/zcl_abapgit_objects_check.clas.abap

```diff
@@ -108,7 +123,11 @@ CLASS zcl_abapgit_objects_check IMPLEMENTATION.
 
 
 * make sure to get the current status, as something might have changed in the meanwhile
-    lt_overwrite = warning_overwrite_find( ct_results ).
+    IF iv_reset_all IS INITIAL.
+      lt_overwrite = warning_overwrite_find( ct_results ).
+    ELSE.
+      lt_overwrite = warning_overwrite_find_all( ct_results ).
+    ENDIF.
```

Comment:
can we calculate a status column in `lt_overwrite` instead of having 2 sets of logic at this level?

and then in ZCL_ABAPGIT_SERVICES_REPO=>GUI_DESERIALIZE it can choose if some of the choices should be left out

Reply:
if `lt_overwrite` can contain everything, then perhaps it should be renamed

Reply:
yes to status field (see below). renaming would quite a bit of work since `overwrite` is used a lot (vars, methods, ls_check, popups). i rather spend some time combining the `delete objects` popup with the `overwrite` so there's one selection of objects to be changed.

Reply:
maybe we add an "action" column to the popup that describes what happens to the object (delete, delete & recreate, overwrite local).

Reply:
okay 👍